<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationPanel</class>
 <widget class="QWidget" name="OperationPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_xsub">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>142</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yadd">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>14</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zadd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>14</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zsub">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>270</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clockwise">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>14</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>顺</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ysub">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>270</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_xadd">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>142</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_input">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>142</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border-radius: 10px;
border: 1px solid #CCD7E5;

font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 34px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_anticlockwise">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>104</width>
     <height>104</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>104</width>
     <height>104</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:enabled {
background: rgba(55,173,186,0.15);
border-radius: 10px;
border: 1px solid #37ADBA;
}

QPushButton:disabled {
background: #F2F7FB;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>逆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
